views/articles/new.ejs

```diff
@@ -0,0 +1,13 @@
+<% include ../partials/header.ejs %>
+
+<h1>Submit a New Article</h1>
+
+<form action="/articles" method="POST">
+	<input type="text" name="title" placeholder="title">
+	<input type="text" name="body" placeholder="body">
+	<input type="submit">
+</form>
+
```

Comment:
add the footer partial link here, it will add the copy rite and close the body and html tags opened when you added the header partial